Udemy Scraper/fetcher.py

```diff
@@ -0,0 +1,77 @@
+import requests
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import time
+import sqlite3
+from sqlite3 import Error
+
+# Function to connect to the SQL Database
+def sql_connection():
+    try:
+        con = sqlite3.connect('udemyDatabase.db')
+        return con
+    except Error:
+        print(Error)
+
+# Function to create table 
+def sql_table(con):
+    cursorObj = con.cursor()
+    cursorObj.execute("CREATE TABLE IF NOT EXISTS courses(title text, description text, instructor text,current_price INTEGER, original_price INTEGER, rating REAL, hours REAL, lectures INTEGER)")
+    con.commit()
+
+# Call functions to connect to database and create table
+con = sql_connection()
+sql_table(con)
+
+# Function to insert into table
+def sql_insert(con, entities):
+    cursorObj = con.cursor()    
+    cursorObj.execute('INSERT INTO courses(title, description, instructor, current_price, original_price, rating, hours, lectures) VALUES(?, ?, ?, ?, ?, ?, ?, ?)', entities)    
+    con.commit()
+ 
+
+# Get chrome driver path
+driver_path = input("Enter chrome driver path")
+
+# Get input for course category to scrape
+category = input("Enter course category:")
```

Comment:
Can you display some of the categories that are available on Udemy? A user like me is not aware of different options in category. Display them as a list to the user. Also this minor change:

```suggestion
category = input("Enter course category: ")
```